CHANGELOG.md

```diff
@@ -12,6 +12,13 @@
   each slot in an epoch) to an epoch/index file pair. Consolidation happens on-the-fly in a
   background process.
 
+- #### Add bouncing and throttling to the API
+  Previously, exchanges could accidentally overload their wallet servers. We
```

Comment:
there was a throttling in v0 per handler https://github.com/input-output-hk/cardano-sl/blob/develop/wallet/src/Pos/Wallet/Web/Methods/Payment.hs#L80